modules/nf-core/gnu/sort/main.nf

```diff
@@ -0,0 +1,50 @@
+process GNU_SORT {
+    tag "${meta.id}"
+    label "process_low"
+
+    conda "conda-forge::coreutils=9.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+    'https://depot.galaxyproject.org/singularity/ubuntu:20.04' :
+    'ubuntu:20.04' }"
+
+    input:
+    tuple val(meta), path(file)
+
+    output:
+    tuple val(meta), file( "*.sorted" )   , emit: sorted
+    path "versions.yml"                    , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args        = task.ext.args     ?: ''
+    def output      = task.ext.prefix   ?: "${meta.id}.txt"
+    output_file     = "${output}.sorted"
```

Comment:
I would tackle the `.sorted` part with the `task.ext.prefix`, or even use a `task.ext.suffix` for it.

Reply:
Would this be a better implementation?
```
    output:
    tuple val(meta), file( "${output_file}" )   , emit: sorted
    path "versions.yml"                         , emit: versions

    when:
    task.ext.when == null || task.ext.when

    script:
    def args        = task.ext.args     ?: ''
    def prefix      = task.ext.prefix   ?: "${meta.id}"
    def suffix      = task.ext.prefix   ?: "${input.extension}"
    output_file     = "${prefix}.${suffix}"
    def VERSION     = "9.1"             // WARN: Version information not provided by tool on CLI. Please update this string when bumping container versions.

    """
    sort ${args} ${input} > ${output_file}

    cat <<-END_VERSIONS > versions.yml
    "${task.process}":
        coreutils: $VERSION
    END_VERSIONS
    """

```

All works as expected locally

Reply:
it should be this:
`suffix      = task.ext.suffix   ?: "${input.extension}"`

Reply:
but yeah, that's perfect

Reply:
That's what I meant.

Nice, all sorted and committed now.